drivers/serial/uart_b91.c

```diff
@@ -341,6 +345,9 @@ static void uart_b91_poll_out(const struct device *dev, uint8_t c)
 
 	uart->data_buf[data->tx_byte_index] = c;
 	data->tx_byte_index = (data->tx_byte_index + 1) % ARRAY_SIZE(uart->data_buf);
+
+	while (!(uart->txrx_status & UART_TXRX_STATUS_TX_DONE)) {
+	};
```

Comment:
Unnecessary semicolon?

Reply:
Same file L343 (no changes applied)
```
	while (uart_b91_get_tx_bufcnt(uart) >= UART_TX_BUF_CNT) {
	};
```
file is already formatted by this way.

Reply:
both should be fixed then

Reply:
fixed